frontend/src/components/Dropdown.tsx

```diff
@@ -6,7 +6,7 @@ const LlmDropdown: React.FC<DropdownProps> = ({ onSelect, isDisabled }) => {
   const handleChange = (selectedOption: OptionType | null | void) => {
     onSelect(selectedOption);
   };
-  const allOptions = useMemo(() => ['Diffbot', 'Gemini Pro'], []);
+  const allOptions = useMemo(() => ['Diffbot', 'OpenAI GPT 4'], []);
```

Comment:
we should just send LLM to the backend and that uses that it's configuration to set the concrete LLM on the Source node.